modules/infinite-scroll/infinity.js

```diff
@@ -102,22 +107,71 @@
 		}
 
 		// Initialize any Core audio or video players loaded via IS
-		this.body.bind( 'post-load', { self: self }, self.initializeMejs );
+		this.body.addEventListener( 'post-load', self.initializeMejs );
+	};
+
+	Scroller.prototype.triggerEvent = function( eventName, el, data ) {
```

Comment:
**Blocker:** Events [need to be triggered](https://github.com/Automattic/jetpack/pull/14886#issuecomment-601156200) in a backwards-compatible fashion. See [this commit](https://github.com/Automattic/jetpack/pull/14886/commits/3bf681adeb348276cdafd7fe3a3fa0423855e82e) for sample implementation.

Reply:
Resolved in fda8991.